### PR TITLE
[WIP] Bananium tree and also includes a updated description for robust harvest

### PR DESCRIPTION
**Pull request by ma44**
This will be a hard plant to grow and reap rewards from, as this plant requires 2 mutations and 100 potency to get the thing you want from growing this plant. This will also have a super low endurance and lifespan so it should take damage after it's done harvesting when you grow it for the first time, and will require lots of water and stuff

The bananium tree is a meme tree. It is mutated from bluespace or mime bananas (which are mutated from bananas) This plant has a rarity of 50 and when it reaches 100 potency, the trash of the banana becomes a bananium sheet (uses some sort of magic)

credit to @GunHog for idea

TODO list: Sprites for seed packets, plant and product
A golden banana peel if it's less than 100 potency
Compiling and testing

### CODE REVIEW

**Comment by Chocobro** [comment]
:-1:
literally only adding this so he can powergame

**Comment by ma44** [comment]
New update: Makes the bananium plant have 1 endurance and 1 lifespan; meaning you will have to keep constant attention for when it's growing or it will die

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -109,6 +109,39 @@
...
+	seed = /obj/item/seeds/banana/bananium
+	name = "Bananium Banana"
+	desc = "If this plant has enough potency, there will be something special inside...."
+	icon_state = "banana"
+	item_state = "banana"
+	trash = /obj/item/weapon/grown/bananapeel
+	filling_color = "#FFFF00"
+	bitesize = 5
+	
+/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/add_juice()
```
Comment (GunHog) [inline code/modules/hydroponics/grown/banana.dm]: Try this:
`/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/New()`
`	materials = list(MAT_BANANIUM = potency*20)`
`	..()`

**Comment by GunHog** [comment]
Instead, you should set the material amount based on the potency, with 100 potency having a full sheet. You grind the bananas in a crusher or destructive analyzer to get the bananium.

**Comment by ExcessiveUseOfCobblestone** [comment]
Ma44 is the steelpoint of hydroponics.

**Comment by ma44** [comment]
Slaps in @GunHog equation on top; will go and fix possible issues later. 
Also made the plant a bit more robust

File: code/modules/hydroponics/grown/banana.dm [hunk 2]
```diff
@@ -137,11 +137,10 @@
 	filling_color = "#FFFF00"
 	bitesize = 5
 	
-/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/add_juice()
+/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/New()
```
Comment (GunHog) [inline code/modules/hydroponics/grown/banana.dm]: I have not tested this btw, so make sure you do! Compile it, run the code, throw it in the DA and see if you get bananium!

**Pull request closed without merging**